packages/hydrooj/src/loader.ts

```
@@ -43,29 +44,20 @@ if (process.env.NIX_PROFILES) {
-    if (config === false) return;
-    if (config === true) config = undefined;
-    config ??= {};
-    const schema = plugin['Config'] || plugin['schema'];
-    if (schema && plugin['schema'] !== false) config = schema(config);
-    return config;
-}
-
 export class Loader extends Service {
-    public state: Record<string, any> = Object.create(null);
+    public state: Record<string, EffectScope> = Object.create(null);
     public config: {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)